### PR TITLE
prevent signals to be sent twice to child process

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
fix signal forwarding to child `com.docker.cli` process
child process is ran in a distinct process group so it won't receive the same signal as parent process from a Ctrl+C in terminal
also filter out some signal as copy/paste from docker/cli [ForwardAllSignals](https://github.com/docker/cli/blob/74a6e2cf627be262d1ba2d55e0f66856038fddd8/cli/command/container/signals.go#L16)

**Related issue**
close https://github.com/docker/compose-cli/issues/1151

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Do you think we should move the `ForwardAllSignals` utility to a different location inside docker/cli so that it can be consumed here?

**Comment by ndeloof** [comment]
We already have a list of func we'd like to reuse from docker/cli to avoid such duplications

**Comment by thaJeztah** [comment]
> We already have a list of func we'd like to reuse from docker/cli to avoid such duplications

👍 happy to discuss; we should probably look what's the most suitable location (docker/cli, or if it's more related to the API client, perhaps in moby/moby/client (e.g.))

File: cli/mobycli/exec.go [hunk 1]
```diff
@@ -117,6 +132,7 @@ func RunDocker(childExit chan bool, args ...string) error {
 				if isRuntimeSig(sig) {
 					continue
 				}
+				fmt.Println("forward signal to process " + sig.String())
```
Comment (thaJeztah) [inline cli/mobycli/exec.go:135]: Was this for debugging? Do we want this to print output?

**Comment by ndeloof** [comment]
Closing, doesn't work :'(

**Pull request closed without merging**